storybook/decorators/with-global-css.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import basicStyles from '../global-basic.lazy.scss';
+import wordPressStyles from '../global-wordpress.lazy.scss';
+
+/**
+ * A Storybook decorator to inject global CSS.
+ *
+ * This helps test whether our components have sufficient styles to
+ * hold up in various CSS environments.
+ */
+
+const config = {
+	none: {
+		lazyStyles: [],
+		externalStyles: [],
+		classes: [],
+	},
+	basic: {
+		lazyStyles: [ basicStyles ],
+		externalStyles: [],
+		classes: [],
+	},
+	wordpress: {
+		lazyStyles: [ wordPressStyles ],
+		externalStyles: [
+			// wp-admin loads "global" stylesheets which contain some broadly scoped styles
+			// that affect wp-components
+			'https://wordpress.org/gutenberg/wp-admin/css/common.min.css',
+			'https://wordpress.org/gutenberg/wp-admin/css/forms.min.css',
```

Comment:
Do you think that we may end up in an "out-of-sync" situation between the version of the styles hosted on `wordpress.org` and the styles that part of a given PR ?

Reply:
And we could be in trouble either way since components need to work with all versions of WP 😂

I don't expect there to be frequent changes to the kind of broadly scoped styles that are affecting us, but if we do encounter issues we could consider the Trac or [GitHub-hosted mirror](https://github.com/WordPress/WordPress/blob/master/wp-admin/css/forms.css) as a source.

Reply:
Fair point 😅 Let's keep it as it is, and make changes if necessary